src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -3150,6 +3150,18 @@ internal bool ShouldSuppressNullableAnnotations(Symbol definition)
             return true;
         }
 
+        private bool HasAllowedNullableAssemblyOptOut(AssemblySymbol assembly)
```

Comment:
Or "HaveNullableOptOutForAssembly"

---
In reply to: [134013697](https://github.com/dotnet/roslyn/pull/21309#discussion_r134013697) [](ancestors = 134013697)